backend/oasst_backend/prompt_repository.py

```diff
@@ -104,8 +106,8 @@ def insert_message(
             review_result=review_result,
         )
         self.db.add(message)
-        self.db.commit()
-        self.db.refresh(message)
+
+        # self.db.refresh(message)
```

Comment:
could we reactivate this (e.g. sever-side `created_date`)

Reply:
if the intention is to get the database-side value for all the attributes of Message then this is achieved by the FLUSH in the decorator. It turns the lazy loading to eager loading. Message will contain all the values before it is returned to the caller